### PR TITLE
Translate some strings into russian language

### PR DESCRIPTION
**Pull request by cdefgah**
Translated some strings into russian language. Fixed some incorrect translations.

### CODE REVIEW

File: translations/tiled_ru.ts [hunk 1]
```diff
@@ -398,17 +398,17 @@
...
         <location line="+28"/>
         <source>Error mapping tile id %1.</source>
-        <translation>Ошибка при создании карты для тайла с ID %1.</translation>
+        <translation>Ошибка при создании карты для тайла с id %1.</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>This seems to be no valid flare map. A Flare map consists of at least a header section, a tileset section and one tile layer.</source>
-        <translation>Файл не соответствует спецификации Flare-карты. Flare карта состоит по крайней мере из одной секции заголовка, набора тайлов и слоя тайлов.</translation>
+        <translation>Файл не соответствует спецификации flare-карты. Flare карта состоит по крайней мере из одного раздела заголовка, раздела с набором тайлов и одного слоя тайлов.</translation>
```
Comment (bjorn) [inline translations/tiled_ru.ts]: Why change "Flare" to lowercase?

File: translations/tiled_ru.ts [hunk 2]
```diff
@@ -560,72 +560,72 @@
     <message>
         <location filename="../src/tiled/mainwindow.ui" line="+49"/>
         <source>&amp;File</source>
-        <translation>&amp;Файл</translation>
+        <translation>Файл</translation>
```
Comment (bjorn) [inline translations/tiled_ru.ts]: Why remove the accelerator?

File: translations/tiled_ru.ts (hunk 1 above)
Comment (cdefgah) [reply translations/tiled_ru.ts]: Sorry, that was my fault. I will redo the translation and will make new PR. You may decline this PR.

File: translations/tiled_ru.ts (hunk 2 above)
Comment (cdefgah) [reply translations/tiled_ru.ts]: Please decline this PR, I will redo the translation and will make new PR.

File: translations/tiled_ru.ts (hunk 1 above)
Comment (cdefgah) [reply translations/tiled_ru.ts]: I have done new pull request, where all strings were translated to Russian language. Please check.

**Comment by cdefgah** [comment]
I closed this PR, because I just have done another corrected PR.

**Pull request closed without merging**